pkg/scheduler/framework/plugins/registry.go

```diff
@@ -73,3 +73,16 @@ func NewInTreeRegistry() framework.Registry {
 		defaultbinder.Name:                         defaultbinder.New,
 	}
 }
+
+// IsCSIRelatedPlugins returns true if the specific plugin name is a CSI related plugin.
+func IsCSIRelatedPlugins(name string) bool {
+	switch name {
+	case nodevolumelimits.CSIName,
+		nodevolumelimits.AzureDiskName,
+		nodevolumelimits.GCEPDName,
+		nodevolumelimits.EBSName,
+		nodevolumelimits.CinderName:
```

Comment:
These constants are defined in pkg/scheduler/framework/plugins/nodevolumelimits/non_csi.go, so I assumed they're not CSI related?

Reply:
Yeah, these plugins defined in non_csi.go, but all these plugins use a `csiNodeLister` to check whether the volume is really non-csi or not, see [this](https://github.com/kubernetes/kubernetes/blob/53f3699cf936b14e04b8533e51e8e7ffe0eeb7ef/pkg/scheduler/framework/plugins/nodevolumelimits/non_csi.go#L232). So I add these plugins too for safety's sake.

Reply:
I will delegate the review to @davidz627 as he knows more details here.

Reply:
`VolumeBinding` plugin needs to access CSINode informer too